resources/packs/gcp/gcp.lr

```diff
@@ -556,43 +559,234 @@ private gcp.storage.bucket @defaults("id") {
   iamConfiguration dict
 }
 
-// GCP Cloud SQL
-gcp.sql {
-  //
-  instances() []gcp.sql.instance
+// GCP Cloud SQL Resources
+private gcp.project.sqlService {
+  // Project ID
+  projectId string
+  // The list of Cloud SQL instances in the current project
+  instances() []gcp.project.sqlService.instance
 }
 
 // GCP Cloud SQL Instance
-private gcp.sql.instance @defaults("name") {
-  // Instance name
-  name string
+private gcp.project.sqlService.instance @defaults("name") {
+  // Project ID
+  projectId string
+  // All maintenance versions applicable on the instance
+  availableMaintenanceVersions []string
   // Backend type
   backendType string
-  // Connection name
+  // Connection name of the instance used in connection strings
   connectionName string
-  // Database version
+  // Creation timestamp
+  created time
+  // The current disk usage of the instance in bytes. This is deprecated, monitoring should be used instead
+  currentDiskSize int
+  // The current database version running on the instance
```

Comment:
```suggestion
  // Current database version running on the instance
```

We're moving to stop beginning the field help with "The" :) Sorry it's a pain!

Reply:
For new resources I am already taking that into account. Next PRs will have less docs issues 😛

Reply:
Thanks for your patience!